### DISCUSSION TITLE
Seemingly ChatGPT integration with Smart Connect can only be used with ChatGPT Plus subscription, any ideas?

### DISCUSSION THREAD

**Discussion by j-peeters** [brianpetro/obsidian-smart-connections#639] [Q&A] [answered]
I was following the following video by @brianpetro https://www.youtube.com/watch?v=TDUWvOc7xJo. But I stumbled upon a road block: OpenAI locked the 'Create GPTs' behind the ChatGPT Plus subscription. So to get this to work like in the video, I would need to pay $20/month, and I cannot afford that currently. 
I have access to the API though and can use it via credits. 
Is there a way to make the connection by making use of the OpenAI API. Or another way? Thanks!

**Comment by brianpetro** [comment, marked as answer]
Hi @j-peeters 

> Is there a way to make the connection by making use of the OpenAI API. Or another way? Thanks!

Currently, there is no way to use it via the API. However, I'm testing this locally, and it will be released to the supporter early-access program shortly, which also does have a cost. But it will be released for free after it makes it through that last phase.

That said, if used with the API, the API cost will quickly outpace the cost of a ChatGPT subscription. I discuss this more here https://www.youtube.com/watch?v=ztHhuufvfVs&t=577s
 
🌴

**Comment by j-peeters** [comment]
Hi @brianpetro Thanks for your swift response and explanation. The video you're pointing me to helps a lot in understanding the interaction between your apps and OpenAI and the costs. I'm low on funds atm, so I'll have to stay with the local semantic version for now, but I'll keep an eye on developments. Thanks for developing this tool set, I find it incredibly exciting.

**Reply by brianpetro** [reply]
Thanks for the kind words. I understand and can relate to anyone who is unable to financially support the project. I've been there!

Accessibility is a key part of my mission with Smart Connections, so you can expect more and more to be available at low-to-no-cost, while I continue to work on bleeding-edge features with the help of those who are able to financially support the necessary research & development into those new features in exchange for early-access.

I also hope to provide more well-deserving people with access to features at cost. However, I'm personally not able to financially support that endeavor myself at the moment. But I believe it is something that is important for the supportive community I'm working towards building around Smart Connections 😊🌴

**Reply by j-peeters** [reply]
I understand that all too well. It surely takes a lot of your time, so funds are needed. 
When I have those, I'll start supporting your work! Meanwhile, I follow it with great interest! 🌅
